apps/desktop/src/lib/branch/BranchLane.svelte

```diff
@@ -44,7 +44,9 @@
 	// BRANCH SERVICE
 	const prService = createGitHostPrServiceStore(undefined);
 	$effect(() =>
-		prService.set(upstreamName ? $gitHost?.prService(baseBranchName, upstreamName) : undefined)
+		prService.set(
+			upstreamName ? $gitHost?.prService(baseBranchName ?? '', upstreamName) : undefined
```

Comment:
Maybe do `upstreamName && baseBranchName` instead of passing empty string?

Reply:
Yeah good idea, fixed